src/odata-to-abstract-sql.ts

```diff
@@ -1168,11 +1171,16 @@ export class OData2AbstractSQL {
 							return ['Not', bool];
 						}
 						case 'in': {
```

Comment:
The changes to `in` should be reverted right since we're no longer going via this path? And that way this module can remain backwards compatible.